content/digital-service-standard/11-measure-performance/index.yml

```diff
@@ -14,10 +14,6 @@ main:
   - how.md
   - guidance-head.md
   - guidance.md
-  - reading-head.md
```

Comment:
How come we're deleting `reading-head.md` and `reading.md` lines as well @zedzedzed?

Reply:
I thought that's what you wanted in https://github.com/govau/service-manual/pull/771#discussion_r242402956 ? Ie delete the partial and the heading.

Reply:
Sorry I wan't clearer. Only the `guidance.md` and `guidance-head.md` partials and the references to those partials in the` index.yml`.

Reply:
Oh good pickup! I removed the wrong ones. Should be back now.